02/app.js

```diff
@@ -1 +1,40 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+/*Todo List*/
+
+/*1.Wyszukanie linków z atrybutami data-url.*/
+
+const links = document.querySelectorAll('a');
+    console.log (links);
+
+const arr = Array.from(links);
+    console.log (arr);
+
+
+
+
+
+/*2.pobranie zawartości tego atrybutu do zmiennej.*/
+
+const dataAtrribute = arr[0].dataset.url;
```

Comment:
Pobierasz, a potem przypisujesz tj. `arr[0].setAttribute('href', arr[0].dataset.url)`

PS. Dobrze by było użyć pętli `for` :)